__tests__/fixtures/react-native/fabricRefRewrites/reactProduction/output.js

```diff
@@ -337,18 +360,19 @@ exports.cloneElement = function (a, b, e) {
   const isTurboModuleEnabled = global.RN$Bridgeless || global.__turboModuleProxy != null;
   if (isTurboModuleEnabled && Platform.OS === 'ios') {
     if (
-      a.$$typeof &&
-      (a.$$typeof.toString() === 'Symbol(react.forward_ref)' ||
-        a.$$typeof.toString() === 'Symbol(react.element)' ||
-        a.$$typeof.toString() === 'Symbol(react.transitional.element)')
+      isReact19Plus() ||
+      (a.$$typeof &&
+        (a.$$typeof.toString() === 'Symbol(react.forward_ref)' ||
+          a.$$typeof.toString() === 'Symbol(react.element)' ||
+          a.$$typeof.toString() === 'Symbol(react.transitional.element)'))
     ) {
       if (d) {
         const propContainsFSAttribute = SUPPORTED_FS_ATTRIBUTES.some(fsAttribute => {
-          if (!!props[fsAttribute]) {
+          if (!!d[fsAttribute]) {
             if (fsAttribute === 'fsAttribute') {
-              return typeof props[fsAttribute] === 'object';
```

Comment:
The variable `props` doesn't exist. This is an example for the dynamic variable name detection bugfix.